app/components/windows/base.py

```diff
@@ -85,7 +86,10 @@ def explorer(self):
 
     def closeEvent(self, event):
         try:
-            rmtree("./poricom_cache")
+            if platform == "linux":
+                rmtree("/tmp/poricom_cache")
+            else:
+                rmtree("./poricom_cache")
```

Comment:
Instead of checking the current platform every time, create a variable `PORICOM_CACHE` in `utils/constants`.